terraform/staging-paas/terraform.tfvars

```diff
@@ -191,5 +191,18 @@ credentials = {
       hosted_graphite_host = "carbon.hostedgraphite.com"
     }
   }
+  metric_exporter = {
+    pay_low_pass_secrets = {
+    }
+
+    static_values = {
+      cf_api_endpoint  = "https://api.cloud.service.gov.uk"
+      cf_username      = "placeholder"
+      cf_password      = "placeholder"
+      statsd_endpoint  = "statsd.hostedgraphite.com:8125"
+      statsd_prefix    = "placeholderapikey.paas.system"
```

Comment:
I added the hosted graphite api key to pay-low-pass, see carbon-relay for an example:
https://github.com/alphagov/pay-omnibus/blob/d4471c0a1fc3bf4300f00daa7ca60127ede51b18/terraform/staging-paas/terraform.tfvars#L181

`hosted_graphite_api_key = "hosted_graphite/prod/api_key"`

Reply:
Thank you 👍